dpm/client/__init__.py

```diff
@@ -9,21 +9,30 @@
 class DpmException(Exception):
     pass
 
+
+class ConfigError(Exception):
+    """ The configuration passed to the client is malformed """
+    pass
+
+
 class Client(object):
 
-    def __init__(self, data_package_path='', config_path="~/.dpm/config", click=None):
+    def __init__(self, config, data_package_path='', click=None):
```

Comment:
search a datapackage on the registry server

Searching from the client is also part of the scope for the project, so yes.